lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -102,9 +103,44 @@
           "type": "array",
           "items": {
             "type": "cid-link",
-            "description": "List of new blobs (by CID) referenced by records in this commit."
+            "description": "DEPRECATED -- will soon always be empty. List of new blobs (by CID) referenced by records in this commit."
           }
         },
+        "prevData": {
+          "type": "cid-link",
+          "description": "The root CID of the MST tree for the previous commit from this repo (indicated by the 'since' revision field in this message). Corresponds to the 'data' field in the repo commit object. NOTE: this field is effectively required for the 'inductive' version of firehose."
+        },
+        "time": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Timestamp of when this message was originally broadcast."
+        }
+      }
+    },
+    "resync": {
+      "type": "object",
+      "description": "Updates the repo to a new state, without necessarily including that state on the firehose. Used to recover from broken commit streams, data loss incidents, or in situations where upstream host does not know recent state of the repository.",
+      "required": [
+        "seq",
+        "did",
+        "commit",
+        "time"
+      ],
```

Comment:
I'd lean towards also including `rev`.

Reply:
I added `rev`, and also `data`. I don't think we have consensus on `data` (@dholms didn't include it in his most recent proposal). I could go either way on it... `rev` is the more important I think.